src/test/java/com/easypost/EasyPostTest.java

```diff
@@ -1002,7 +1000,7 @@ public void testPaymentLogReport() throws EasyPostException {
     public void testWebhookCRUD() throws EasyPostException {
         // Define request params
         Map<String, Object> paramMap = new HashMap<String, Object>();
-        String url = "https://example.com/" + UUID.randomUUID().toString();
```

Comment:
Does `UUID.randomUUID()` already return a string?

Reply:
It returns a UUID object, which inherently knows how to convert to a string when added to another string. So the explicit conversion is unnecessary and the linter removed it.